plugin.xml

```diff
@@ -62,7 +62,7 @@
         <source url="https://github.com/CocoaPods/Specs.git"/>
       </config>
       <pods use-frameworks="true">
-        <pod name="google-cast-sdk" spec="~> 4.4.1" />
+        <pod name="google-cast-sdk" spec="~> 4.4.4" />
```

Comment:
This won't break the Java implementation will it?

Reply:
This would not affect java/android at all